modules/processes/prepare_faust_data.nf

```diff
@@ -46,11 +46,18 @@ process prepareFAUSTData {
             convert_legacy_gs("${input_gating_set_directory_channel}", converted_gating_set_directory_path)
             gating_set <- try(flowWorkspace::load_gs(converted_gating_set_directory_path))
         }
-
-        active_channels_rds_object <- readRDS("${active_channels_path_channel}")
-        channel_bounds_rds_object <- readRDS("${channel_bounds_path_channel}")
-        supervised_list_rds_object <- readRDS("${supervised_list_path_channel}")
-
+        active_channels_rds_object<-NULL
```

Comment:
The `null` test below requires these to be defined and set to null here.

Reply:
Good catch and great fix!